### DISCUSSION TITLE
inside wall temperature spatial statistics - simulation reports error

### DISCUSSION THREAD

**Discussion by grrobic** [firemodels/fds#12711] [Q&A] [answered]
Hello,

I am having trouble setting up spatial statistics inside wall temperature device. Simulation stops few seconds in with error "DEVC 11 must be associated with a heat-conducting surface". I have MATL with thermal parameters set up and associated with SURF of wall for which I would like to measure inside wall temperature. The discrete inside wall temperature DEVC reports no problem, however, spatial statistics DEVC does not work in my simulation. Is my volume for spatial statistics set up incorrectly? Can someone please help me find the problem in my fds file (attached below)?

Thank you.

[osnovni_burner_HRRPUA_600.txt](https://github.com/firemodels/fds/files/14807725/osnovni_burner_HRRPUA_600.txt)

**Comment by mcgratta** [comment, marked as answer]
Try adding `SURF_ID` to the `DEVC` line

**Reply by grrobic** [reply]
Thank you! This is what was missing in my fds file.
